### ISSUE TITLE
Request: reconnect

### ISSUE THREAD

**Issue by pbb72** [HeidiSQL/HeidiSQL#653]
My SQL Server is closing the connection for HeidiSQL after a certain amount of time has passed. It would be very nice if there was a quick way to rebuild the same connection.

## Steps to reproduce this issue

1. Open a connection to a database server.
2. Wait a long time until the connection is automatically broken by the server.

## Current behavior

Now I have to perform the following steps to reconnect:

1. Open the Session Manager.
2. Find and select the correct session.
3. Open the connection by pressing "OK".
4. Find two connections in the left pane, one gray (the closed one) and one black (the currently open one).

## Possible solution

1. Press a simple button to re-connect. Possibly combined with the button that closes the selected database connection, since that button has no purpose when the connection is already closed.

Finished.

## Environment

* **HeidiSQL version:** HeidiSQL Portable 10.1.0.5464
  
* **Database system and version:** Microsoft SQL Server (TCP/IP)
  
* **Operating system:** Microsoft Windows 10 Enterprise

**Comment by ansgarbecker** [comment]
HeidiSQL is desiged to reconnect seamlessly, as soon as you need that. For instance when you are about to execute a query, or when you press F5 to refresh the current page. There is no need to go to the session manager in such cases.

**Comment by pbb72** [comment]
When I do an F5 refresh after some hours, I get the following message:

> SQL Error (0): Operation is not allowed when the object is closed
> unknown

![image](https://user-images.githubusercontent.com/1436366/59015367-1baab400-883f-11e9-9241-67d34c03e11a.png)

Even better, when I go to File > Connect To, and select the current connection, the application crashes.

![image](https://user-images.githubusercontent.com/1436366/59015596-a4c1eb00-883f-11e9-8712-1af20be636ea.png)

**Comment by ansgarbecker** [comment]
Yes, these are bugs in my implementation for MSSQL connections. If instead of these errors/crashes, HeidiSQL would just do what it is supposed to do, then I guess this is a valid reconnection logic, isn't it?

**Comment by pbb72** [comment]
Yes, that is why I suggested a button to reconnect to the database server.
